src/sasctl/pzmm/pickleModel.py

```diff
@@ -3,14 +3,14 @@
 
 
 # %%
-import os
+from pathlib import Path
 
 import pickle
 
 # %%
 class PickleModel():
     
-    def pickleTrainedModel(self, trainedModel, modelPrefix, pPath=os.getcwd()):
+    def pickleTrainedModel(trainedModel, modelPrefix, pPath=Path.cwd()):
```

Comment:
def pickleTrainedModel(self, trainedModel, modelPrefix, pPath=Path.cwd()):

Reply:
Fixed in [07ad6ce](https://github.com/sassoftware/python-sasctl/pull/76/commits/07ad6ce2532e24b1ef56e54c5b7ec572621f1fb0)